src/Tools/AnalyzerRunner/IncrementalAnalyzerRunner.cs

```diff
@@ -10,6 +10,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
```

Comment:
I think we can delete this entire file and even the `IncrementalAnalyzerNames` option